test/integration/kubernetes_deploy_test.rb

```diff
@@ -1078,4 +1080,14 @@ def test_not_apply_resource_can_be_pruned
       /The following resources were pruned: #{pod_disruption_budget_matcher}/
     ])
   end
+
+  private
+
+  def expected_daemonset_pod_count
+    nodes = kubeclient.get_nodes
+    return 1 if nodes.one?
+    nodes.count do |node|
+      !node.metadata.labels.to_h.keys.include?(:"node-role.kubernetes.io/master")
```

Comment:
dind gives you a multi node setup?

Reply:
Yep! Two nodes by default. Here's the repo for it FYI: https://github.com/kubernetes-sigs/kubeadm-dind-cluster